files/ko/web/javascript/reference/global_objects/object/getownpropertydescriptors/index.html

```diff
@@ -0,0 +1,107 @@
+---
+title: Object.getOwnPropertyDescriptors()
+slug: Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptors
+tags:
+- JavaScript
+- Method
+- Object
+- Polyfill
+browser-compat: javascript.builtins.Object.getOwnPropertyDescriptors
+---
+<div>{{JSRef}}</div>
+
+<p><code><strong>Object.getOwnPropertyDescriptors()</strong></code> 메서드는 주어진 객체의 모든 속성들의 설명자(descriptor)들을 반환합니다.</p>
+
+<div>{{EmbedInteractiveExample("pages/js/object-getownpropertydescriptors.html")}}</div>
+
+<h2 id="Syntax">구문</h2>
+
+<pre class="brush: js">Object.getOwnPropertyDescriptors(<var>obj</var>)</pre>
+
+<h3 id="Parameters">매개변수</h3>
+
+<dl>
+  <dt><code>obj</code></dt>
+  <dd>가지고 있는 모든 속성들의 설명자를 반환받고 싶은 객체</dd>
+</dl>
+
+<h3 id="Return_value">반환 값</h3>
+
+<p>객체의 모든 속성의 설명자를 가지는 객체를 반환합니다. 매개변수 객체에 속성이 없다면, 빈 객체가 반환됩니다.</p>
+
+<h2 id="Description">설명</h2>
+
+<p>이 메서드를 사용하면, 객체의 모든 속성들에 대한 설명자를 정확하게 확인할 수 있습니다.
+    자바스크립트의 <dfn>속성</dfn>은 문자열로된 이름 또는 심볼, 그리고 속성 설명자로 이루어져 있습니다.
+    속성 설명자 종류와 특성에 대한 자세한 정보는 {{jsxref("Object.defineProperty()")}}에 있습니다..
+</p>
+
+<p><dfn>속성 설명자</dfn> 는 다음과 같은 특징들을 가진 레코드입니다.</p>
+
+<dl>
+  <dt><code>value</code></dt>
+  <dd>속성과 관련된 값입니다. (데이터 설명자만 해당됨)</dd>
+  <dt><code>writable</code></dt>
+  <dd><code>참(true)</code>인 경우, 속성과 관련된 값이 변경될 수 있는 상태입니다. (데이터 설명자만 해당됨)</dd>
+  <dt><code>get</code></dt>
+  <dd>속성에 대한 getter 역할을 하는 함수거나, getter가 정의되지 않은 경우에는 {{jsxref("undefined")}} 입니다.
+    (접근자 설명자만 해당됨)</dd>
+  <dt><code>set</code></dt>
+  <dd>속성에 대한 setter 역할을 하는 함수거나, setter 정의되지 않은 경우에는 {{jsxref("undefined")}} 입니다.
+    (접근자 설명자만 해당됨)</dd>
+  <dt><code>configurable</code></dt>
+  <dd><code>참(true)</code>인 경우, 객체로 부터 속성 설명자가 변경 및 삭제될 수 있는 상태입니다.</dd>
+  <dt><code>enumerable</code></dt>
+  <dd><code>true</code>인 경우, 열거 가능한 상태의 속성입니다.</dd>
+</dl>
+
+<h2 id="Examples">예시</h2>
+
+<h3 id="Creating_a_shallow_clone">얕은 복사하기</h3>
+
+<p>{{jsxref("Object.assign()")}} 메서드는 원본 객체에서 목표 객체로 열거 가능한 속성과 객체의 속성들만 복사하는 반면,
+   이 메서드와 {{jsxref("Object.create()")}} 를 사용하면 어느 두 객체를 얕은 복사에 사용할 수 있습니다.</p>
+
+<pre class="brush: js">Object.create(
+  Object.getPrototypeOf(obj),
+  Object.getOwnPropertyDescriptors(obj)
+);
+</pre>
+
+<h3 id="Creating_a_subclass">하위 클래스 만들기</h3>
```

Comment:
```suggestion
<h3 id="Creating_a_subclass">서브클래스 만들기</h3>
```

![Screen Shot 2021-07-11 at 12 13 49 AM](https://user-images.githubusercontent.com/33975709/125167744-f6868d80-e1dc-11eb-8b37-6961e3a10ff7.png)

`superclass` -> `슈퍼 클래스` / `subclass` -> `하위 클래스` 로 번역하셔서 이 둘 단어가 서로 어색해 [MS Terminology 사이트](https://www.microsoft.com/en-us/language)를 확인해본 결과 두 단어 모두 영어 발음대로 번역하는 것을 확인했습니다. 해당 내용에 맞게 수정하시길 추천드립니다.

Reply:
추천해 주신 바가 더 자연스러울것 같아 내용 반영했습니다.
:)

Reply:
@cos18 혹시 해당 사항들에 대해서 이번 단어 안건에 올려주실 수 있으신가요?